tox.ini

```diff
@@ -67,6 +67,16 @@ commands =
     "{envpython} -m pyspelling --config {toxinidir}/.spellcheck.yml" --spellchecker aspell
 allowlist_externals = sh
 
+[testenv:docs]
+description = docs
+skipsdist = true
```

Comment:
`skipsdist` is only useful in combination with `skip_install`. Otherwise tox still creates a package.
```suggestion
```